### ISSUE TITLE
Can't set the background color when dragged

### ISSUE THREAD

**Issue by jinwon7878** [10play/10tap-editor#180]
I think we're talking about a different problem, and I apologize if I'm still confusing you.

Your solution doesn't solve my problem.
Your solution is a way to change the 'font color' of the editor.
But that's not what I need, because the text color changes just fine without the trick.
What I want is a way to change the 'when dragged, the background color' in the editor.

The 'output image' according to your solution also doesn't have the solution I want: the background color of the dragged part (selection text) is still white. **If the text is white, the text is not visible because it is similar to the background color.**

If this is not resolved, I will continue to work around the issue on my own as before. Until this is resolved, I recommend leaving the issue open to find a good solution.

I think I may have raised the issue confusingly before, so I'll reopen the issue to clarify.

---------------------------------------------------
            
            
              @jinwon7878 - you didnt follow my solution at all

```
      #root div .ProseMirror * {
        color: blue;
      }

      * {
        color: red;
      }

      #root div .ProseMirror {
        background-color: black;
        padding: 0 24px;
        font-size: 15px;
      }
```

output:

![image](https://uploads.linear.app/2729553a-1a7d-4564-8c8e-acf8c04dbbee/6261782a-71c2-45f0-b045-9574976df8e0/8deb0a3d-cec3-43fd-872e-c51a08cc04c6?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzI3Mjk1NTNhLTFhN2QtNDU2NC04YzhlLWFjZjhjMDRkYmJlZS82MjYxNzgyYS03MWMyLTQ1ZjAtYjA0NS05NTc0OTc2ZGY4ZTAvOGRlYjBhM2QtY2VjMy00M2ZkLTg3MmUtYzUxYTA4Y2MwNGM2IiwiaWF0IjoxNzIyODU4Mzc2LCJleHAiOjMzMjkzNDE4Mzc2fQ.fzNSqEdIGuRJQ616LDNl0HFGaXdTwAbVZv7qSeAVw-s)

please notice the:

```
      * {
        color: red;
      }
```

which does the trick here, change the selection color, then:

```

      #root div .ProseMirror * {
        color: blue;
      }
```

Will let you override the color to whatever you want to be the color of the actual text

_Originally posted by @GuySerfaty in https://github.com/10play/10tap-editor/issues/166#issuecomment-2268879058_

**Comment by GuySerfaty** [comment]
you want to change the color of the selected text not the cursor or the background color of the selection,

![image](https://github.com/user-attachments/assets/4871fd72-9000-4075-896a-d6d5e8d28532)

so in the image above the selected text will have different color, I dont think it's possible.
did you ever saw an example of app that modify only the selected text?

**Comment by GuySerfaty** [comment]
@jinwon7878 - that's the reason it's not possible for now:
https://css-tricks.com/almanac/selectors/s/selection/#aa-browser-support

the solution I suggested solve part of your issue but still you wouldn't get full control of the selection till iOS will support that css tag

Im closing this issue for now, if you need anything else please feel free to send email to email@10play.dev

Thank you

**Comment by GuySerfaty** [comment]
@jinwon7878 - ok I think I see what you mean, we will have a solution for it in a bit

**Comment by GuySerfaty** [comment]
Should be fixed on `0.5.17`
@jinwon7878 - can you confirm?
